clang/include/clang/CIR/Dialect/Builder/CIRBaseBuilder.h

```diff
@@ -134,6 +134,17 @@ class CIRBaseBuilderTy : public mlir::OpBuilder {
     return createBinop(lhs, mlir::cir::BinOpKind::Mul, val);
   }
 
+  mlir::Value createSub(mlir::Value lhs, mlir::Value rhs) {
+    return createBinop(lhs, mlir::cir::BinOpKind::Sub, rhs);
+  }
+
+  mlir::Value createNSWSub(mlir::Value lhs, mlir::Value rhs) {
+    auto op = create<mlir::cir::BinOp>(lhs.getLoc(), lhs.getType(),
```

Comment:
We could have only one version of `createSub` and have two default initialized bool params instead.

Reply:
@bcardosolopes 
I have nothing against to rename it, but I thought it's more readable - to have a separate function. So if you think we still need to have only one - I will do it!

Reply:
We can do both approaches, you extend `createSub` with arguments for `bool nsw `and `bool nuw` and make `createNSWSub` call it with one of them `true`.